### PR TITLE
Dcadams/inline analytics last attempt upstream - WIP

### PR DESCRIPTION
**Pull request by dcadams**

### CODE REVIEW

**Comment by dcadams** [comment]
Retest this please

**Comment by dcadams** [comment]
@caesar2164 Would you mind eyeballing the scss.

**Comment by dcadams** [comment]
@kluo @stvstnfrd A lot of this has been review previous but it would be good if you could give it a quick eyeball before I send it upstream.

File: lms/static/sass/course/instructor/_inline_analytics.scss [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            position: relative;
+		    height: 100%;
+		    width: 45px;
+		    margin: 0 15px;
+		    float: left;
+		    
+		    .block {
+		        position: absolute;
+                width: 100%;
+                text-align: center;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_inline_analytics.scss]: what happened with the indents here?

File: lms/static/sass/course/instructor/_inline_analytics.scss [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        margin-left: 50px;
+        border-left: 2px solid #444;
+        border-bottom: 2px solid #444;
+        
+        .y-label {
+            position: absolute;
+		    width: 45px;
+		    top: -8px;
+		    left: 0;
+		    text-align: right;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_inline_analytics.scss]: why are these indented further in than the first line?

File: lms/static/sass/course/instructor/_inline_analytics.scss [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        }
+        
+        .column {
+            position: relative;
+		    height: 100%;
+		    width: 45px;
+		    margin: 0 15px;
+		    float: left;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_inline_analytics.scss]: why are these indented further in than the first line?

File: lms/templates/inline_analytics.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  <a class="instructor-info-action instructor-analytics-action" id="${element_id}_analytics_button" data-location="${location}" data-answer-dist-url="${answer_dist_url}" data-course-id="${course_id}">${_("Staff Analytics Info")}</a>
+</div>
+
+
+<div id="${element_id}_analytics_error_message" style="display: none;">
+  <p><strong></strong></p>
+</div>
+
+<div id="${element_id}_analytics_close" style="display: none;" class="analytics_close_button" href="#">
+  <button class="close">X</button>
```
Comment (caesar2164) [inline lms/templates/inline_analytics.html]: This should probably use Font Awesome

File: lms/static/js/inline_analytics.js [hunk 5]
```diff
@@ -0,0 +1,538 @@
...
+                    tr = $('<tr>');
+                    tdChoice = $('<td class="answer_box">');
+                    tdChoice.attr('title', choiceText[index]);
+                    tdChoice.text(parseInt(index, 10) + 1);
+                    tdDot = $('<td class="answer_box">');
+                    tdDot.addClass(answerClass);
+                    tdDot.append($('<span class="dot">'));
+                    tdCount = $('<td class="answer_box">');
+                    tdCount.text(count);
+                    tdPercent = $('<td class="answer_box">');
```
Comment (caesar2164) [inline lms/static/js/inline_analytics.js]: classes should really always use "-" not "_"

**Pull request closed without merging**